apps/web/src/features/search/components/search-command.tsx

```
@@ -0,0 +1,210 @@
+  CommandSeparator,
+} from '@/components/ui/command';
+import { VibeResult } from './result-items/vibe-result';
+import { UserResult } from './result-items/user-result';
+import { TagResult } from './result-items/tag-result';
+import { ActionResult } from './result-items/action-result';
+import { SearchSuggestions } from './search-suggestions';
+import { useSearchSuggestions } from '../hooks/use-search';
+import { useNavigate } from '@tanstack/react-router';
+
+
+interface SearchCommandProps {
```

ESLint

[error] Delete `⏎`

(prettier/prettier)